include/directx/d3d12.idl

```diff
@@ -5314,7 +5314,9 @@ typedef enum D3D12_BARRIER_SYNC
     D3D12_BARRIER_SYNC_VIDEO_ENCODE                                             = 0x400000,
     D3D12_BARRIER_SYNC_BUILD_RAYTRACING_ACCELERATION_STRUCTURE                  = 0x800000,
     D3D12_BARRIER_SYNC_COPY_RAYTRACING_ACCELERATION_STRUCTURE                   = 0x1000000,
-    D3D12_BARRIER_SYNC_SPLIT                                                    = 0x80000000,
+    // error CS0031: Constant value '-2147483648' cannot be converted to a 'uint' 
+    // D3D12_BARRIER_SYNC_SPLIT = unchecked((int)(0x80000000)),
+    // D3D12_BARRIER_SYNC_SPLIT                                                    = 0x80000000,

```

Comment:
> * `D3D12_BARRIER_SYNC_SPLIT` / `D3D12_BARRIER_ACCESS_NO_ACCESS` use `0x80000000` which result in an odd cast: `D3D12_BARRIER_SYNC_SPLIT = unchecked((int)(0x80000000))`, that's rejected by the C# compiler: `error CS0031: Constant value '-2147483648' cannot be converted to a 'uint'`.

@mikebattista any idea about this from the win32metadata side?

Reply:
https://github.com/microsoft/win32metadata/issues/1369

Reply:
Right, so that's how far you folks have already gotten - would love to see the `.proj` setup even if early!

Reply:
I hit this as well trying to update our SDK metadata to a newer SDK. Will take a look.